### ISSUE TITLE
How can I configure to close this pop-up window? If this feature is not available, is it possible to add it?

### ISSUE THREAD

**Issue by coder-virjay** [carlthome/vscode-git-line-blame#84]
How can I configure to close this pop-up window? If this feature is not available, is it possible to add it?

<img width="730" alt="image" src="https://github.com/user-attachments/assets/0299ef97-44c8-44e2-ac41-7e08eb4f4e32">

**Comment by alexruimy** [comment]
Agreed, the popups are very intrusive. The extension is great but it could really use some more config.
